SPECS/libsemanage/libsemanage.spec

```diff
@@ -102,15 +102,13 @@ ln -sf  %{_libdir}/libsemanage.so.2 %{buildroot}/%{_libdir}/libsemanage.so
 %files python3
 %{python3_sitelib}/*.so
 %{python3_sitelib}/semanage.py*
+%{python3_sitelib}/__pycache__/semanage.cpython*.pyc
```

Comment:
Now I'm getting worried. Is this a merge artifact or you're reverting some of the changes done during Python 3.12 upgrade, because we now generate these files again?

Reply:
Yes, this change was intentional - to remove the temporary fixes I put in place here: https://github.com/microsoft/azurelinux/pull/7858/files#diff-414a05781908e0638662dce8f649e00533b15999686b03c7884b8a047ad259fc
I decided not to bump the release number since this package has subsequently been updated already. I think this should be fine given we are still in the development phase for 3.0